experiment/coverage_utils.py

```diff
@@ -269,6 +343,46 @@ def generate_json_summary(coverage_binary,
     return result
 
 
+# pylint: disable=too-many-locals
+def extract_covered_segments_and_functions_from_summary_json(
+        summary_json_file, benchmark, fuzzer, trial_id, df_container):
+    """Returns the segments and the function given a coverage summary json file.
+    in two separate data frames for reports"""
+    try:
+        coverage_info = get_coverage_infomation(summary_json_file)
+
+        # Extract coverage information for functions.
+        for function_data in coverage_info['data'][0]['functions']:
+            to_append = [
+                benchmark, fuzzer, trial_id, function_data['name'],
+                function_data['count']
+            ]
+            a_series = pd.Series(to_append,
+                                 index=df_container.function_df.columns)
+            df_container.function_df = df_container.function_df.append(
+                a_series, ignore_index=True)
+
+        # Extract coverage information for functions and segments.
+        line_index = 0
```

Comment:
I think these numbers are used elsewhere.
Can you make them filewide constants and reuse?

Reply:
Some same named variables are being used in ```extract_covered_regions_from_summary_json``` but the values assigned are different. Hence the local constants here.